src/libre/dialect/pcre/lexer.lx

```diff
@@ -37,28 +37,26 @@
 '\n' -> $esc;
 '\r' -> $esc;
 '\t' -> $esc;
-'\v' -> $esc;
 
 # perl/unix-ish escapes
 '\a' -> $esc;
 '\c' /./ -> $control;
 
-# perl character class escapes
-'\d' -> $class_digit;
-'\D' -> $class_ndigit;
-'\w' -> $class_word;
-'\W' -> $class_nword;
-'\s' -> $class_space;
-'\S' -> $class_nspace;
+# character type escapes
+# Note: these need to be their own lexemes distinct from named character
+# classes such as /[:digit:]/, as they are allowed in different contexts
+char_type = '\' /[dDhHsSvVwWN]/;
```

Comment:
Unfortunately yes, it breaks the symmetry. I believe I have a note to that effect in the header.

`\N` is supposed to be the opposite of `\n`, since (unless a PCRE flag is set so `.` also matches across lines, off by default) it matches everything except newlines.

The individual `$char__type` patterns are separated in e0561b9.